neural_speed/core/layers/mha_dense.cpp

```diff
@@ -144,6 +187,80 @@ inline __m512 exp_ph_0_1(const __m512 x) {
 }
 #endif
 
+alignas(32) const uint32_t mask8[9][8]{
```

Comment:
why 9?

Reply:
There is 8 dwords in an `_m256`.

And there are 9 values from 0 to 8 (as mask).